docs/docs/references/CLI/rill/project/project.md

```diff
@@ -0,0 +1,30 @@
+## rill project
+
+Manage projects
+
+### Options
+
+```
+      --api-token string   Token for authenticating with the admin API
+      --api-url string     Base URL for the admin API (default "https://admin.rilldata.io")
+      --org string         Organization Name
+```
+
+### Options inherited from parent commands
```

Comment:
I wonder if we should merge options inherited and normal options under same heading ? 
Also should we call them parameters instead of options? 
+cc @begelundmuller

Reply:
In the Cobra `--help` menus, it renders as:
```
Flags:
      --api-token string   Token for authenticating with the admin API
      --api-url string     Base URL for the admin API (default "http://localhost:9090")

Global Flags:
  -h, --help   Print usage
```
Maybe we should mirror that here?

At least, we should use the word "flag" instead of "options".

I think it's helpful to keep the global flags separate from the individual flags, since the flag ordering might be weird otherwise (we currently order flags by expected frequency of use, not alphabetically)

Reply:
sounds good, lets mirror the --help menus here.